src/main/java/uk/gov/hmcts/reform/em/stitching/pdf/PDFMerger.java

```diff
@@ -50,26 +41,32 @@ private static class StatefulPDFMerger {
         private final Logger log = LoggerFactory.getLogger(StatefulPDFMerger.class);
         private final PDFMergerUtility merger = new PDFMergerUtility();
         private final PDDocument document = new PDDocument();
-        private final PDFOutline pdfOutline = new PDFOutline(document);
+        private final PDFOutline pdfOutline;
         private TableOfContents tableOfContents;
         private final Map<BundleDocument, File> documents;
         private final Bundle bundle;
         private static final String BACK_TO_TOP = "Back to index";
         private int currentPageNumber = 0;
         private final File coverPage;
+        private TreeNode<SortableBundleItem> treeRoot;
 
         private StatefulPDFMerger(Map<BundleDocument, File> documents, Bundle bundle, File coverPage) {
             this.documents = documents;
             this.bundle = bundle;
             this.coverPage = coverPage;
+            this.treeRoot = createOutline(bundle);
+
+            this.pdfOutline = new PDFOutline(document, treeRoot);
+
         }
 
         private File merge() throws IOException {
             try {
-                pdfOutline.addBundleItem(bundle.getTitle());
+                pdfOutline.addBundleItem(bundle);
 
                 if (coverPage != null) {
                     try (PDDocument coverPageDocument = PDDocument.load(coverPage)) {
+                        coverPageDocument.getDocumentCatalog().setDocumentOutline(null);
```

Comment:
removes the ouline of cover page EM-4871 so it will not appear when it is merged